python/mdvtools/dbutils/routes.py

```
@@ -1,150 +1,834 @@
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+def register_routes(app, ENABLE_AUTH):
+    from flask import abort, request, jsonify, session, redirect, url_for, render_template, send_file
+    from mdvtools.auth.authutils import update_cache, active_projects_cache, user_project_cache,user_cache, all_users_cache
+    from mdvtools.dbutils.mdv_server_app import serve_projects_from_filesystem, is_valid_mdv_project
+    import os
+    import shutil
+    from mdvtools.mdvproject import MDVProject
+    from mdvtools.project_router import ProjectBlueprint_v2 as ProjectBlueprint
+    from mdvtools.dbutils.dbmodels import User, UserProject
```

Ruff (0.11.9)

`mdvtools.dbutils.dbmodels.UserProject` imported but unused

Remove unused import: `mdvtools.dbutils.dbmodels.UserProject`

(F401)